rpheredia/react-challenge-project-jan-2023
# Upd dependencies

## Changes
Update project dependencies in application.

## Purpose
Upgrade project in order to fix security vulnerabilities in packages. See issue Shift3/react-challenge-project-jan-2023#7

## Approach
Update packages to the latest versions fixes bugs and security vulnerabilities which could be a potential vector for attacks.

## Testing Steps
Login to application and verify that all features work properly.

References: https://catalins.tech/update-project-dependencies-to-the-latest-versions/
https://react.dev/blog/2022/03/08/react-18-upgrade-guide

_If this closes an issue, name it here. If it doesn't, remove this line_
Fixes Shift3/react-challenge-project-jan-2023#7
